scixhelp/orcid-scix/_posts/2016-02-02-claiming-papers.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post_scix
+title: 'Claiming papers in ORCID'
+order: 1
+---
+
+<figure>
+   <img src="/scixhelp/img/orcid-add.gif"  class="img-responsive" alt="a short animated image showing the orcid claiming process">
+</figure>
+
+### Recommended Method
+
+If you care about having your ORCID claims searchable in the SciX.
+
+1. **Make sure you're signed in to SciX.** If you are not signed in already, [sign in here](https://scixplorer.org/user/account/login) or [create an account](https://scixplorer.org/user/account/register).
+
+2. **Authenticate with ORCID.** Click on the ORCID menu in the top navigation bar and follow the instructions.
+
+3. **Enter a bit of extra information in your SciX User Profile.** If you haven't done so, on your [ORCiD Page](https://scixplorer.org/user/orcid) in SciX. Enter in 1) your current academic affiliation and 2) any name variations under which you might have published. Submitting these information will grant SciX permission to track your claims, allowing us to put your data in our system and marking it as trusted so that others can find it.
```

Comment:
Move `Submitting these information will grant SciX permission to track your claims, allowing us to put your data in our system and marking it as trusted so that others can find it.` to step 2 above - we've removed the approval pop-up from the setup page. Let's also change the wording: `Submitting these information` to `This`